_sources/Conditionals/Logicaloperators.rst

```diff
@@ -20,39 +20,133 @@
 Logical operators
 -----------------
 
-There are three **logical operators**: ``and``, ``or``, and ``not``. The
-semantics (meaning) of these operators is similar to their meaning in English.
-For example, ``x > 0 and x < 10`` is true only if ``x`` is greater than 0 *and*
-at the same time, x is less than 10.  How would you describe this in words?  You would say that
-x is between 0 and 10, not including the endpoints.
+There are three **logical operators**: ``and``, ``or``, and ``not``. 
+All three operators take boolean operands and produce boolean values. 
+The semantics (meaning) of these operators is similar to their meaning in English:
+
+* ``x and y`` is ``True`` if both ``x`` and ``y`` are ``True``. Otherwise, ``and`` produces ``False``.
+* ``x or y`` yields ``True`` if either ``x`` or ``y`` is ``True``. Only if both operands are ``False``
+  does ``or`` yield ``False``.
+* ``not x`` yields ``False`` if ``x`` is ``True``, and vice versa.
+
+Look at the following example. See if you can predict the output. Then, Run to see if your predictions were
+correct:
+
+.. activecode:: logop_ex1
 
-``n % 2 == 0 or n % 3 == 0`` is true if *either* of the conditions is true,
-that is, if the number is divisible by 2 *or* divisible by 3.  In this case, one, or the other, or
-both of the parts has to be true for the result to be true.
+   x = True
+   y = False
+   print(x or y)
+   print(x and y)
+   print(not x)
 
-Finally, the ``not`` operator negates a boolean expression, so ``not  x > y``
-is true if ``x > y`` is false, that is, if ``x`` is less than or equal to
-``y``.
+Although you can use boolean operators with simple boolean literals or variables as in the above
+example, they are often combined with the comparison operators, as in this example. Again, before you
+run this, see if you can predict the outcome:
 
-.. activecode:: ac7_3_1
+.. activecode:: chp05_3
```

Comment:
I know its tempting to fix the id, but as a rule once an activecode is published we don't change the identifier.  It messes up any assignments that are in progress.

Reply:
Oops - I've reverted that unintended identifier change.